database/seeders/CleaningStatusSeeder.php

```diff
@@ -14,6 +15,16 @@ class CleaningStatusSeeder extends Seeder
      */
     public function run()
     {
-        //
+        CleaningStatus::insert([
```

Comment:
ook een optie voor als er geen schoonmaak nodig is?

Reply:
Ik heb een status clean toegevoegd